{{cookiecutter.project_slug}}/config/wsgi.py

```diff
@@ -37,10 +33,6 @@
 # file. This includes Django's development server, if the WSGI_APPLICATION
 # setting points here.
 application = get_wsgi_application()
-{% if cookiecutter.use_sentry == 'y' -%}
-if os.environ.get('DJANGO_SETTINGS_MODULE') == 'config.settings.production':
-    application = Sentry(application)
```

Comment:
Remove the WSGI integration as they recommend using a more specific one, when possible:

> It is recommended to use an integration for your particular WSGI framework if available, as those are easier to use and capture more useful information.